src/Tempest/Validation/src/Rules/Custom.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Validation\Rules;
+
+use Attribute;
+use Closure;
+use Tempest\Validation\Rule;
+
+#[Attribute(Attribute::IS_REPEATABLE)]
+final readonly class Custom implements Rule
```

Comment:
Nice PR! The only thing I'm not convinced about it the name. Any ideas for something better? Maybe `Validate` ? Maybe something else?

Reply:
Maybe `CallbackRule` or `Callback`?

Reply:
@innocenzi @brendt you apposed to `LambdaValidation` or is that too obscure??

Reply:
Lambda is not a term used much in PHP-land to refer to closures, so I'm personally not a fan of it 😞

Reply:
Alright, point taken, will be using `Callback` and based on the namespace it should be obvious...